src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/ProductReview.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" ?>
+
+<resources xmlns="https://api-platform.com/schema/metadata"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="https://api-platform.com/schema/metadata https://api-platform.com/schema/metadata/metadata-2.0.xsd"
+>
+    <resource
+        class="Sylius\Component\Core\Model\ProductReview"
+        shortName="ProductReview"
+    >
+        <attribute name="normalization_context">
+            <attribute name="groups">
+                <attribute>product_review:read</attribute>
+            </attribute>
+        </attribute>
+        <attribute name="denormalization_context">
+            <attribute name="groups">
+                <attribute>product_review:update</attribute>
+            </attribute>
+        </attribute>
+        <attribute name="validation_groups">sylius</attribute>
+        <collectionOperations>
+            <collectionOperation name="get" />
+        </collectionOperations>
+        <itemOperations>
+            <itemOperation name="get" />
+            <itemOperation name="delete" />
+            <itemOperation name="put">
+                <attribute name="denormalization_context">
+                    <attribute name="groups">product_review:update</attribute>
+                </attribute>
+            </itemOperation>
+            <itemOperation name="accept_product_review">
+                <attribute name="method">PATCH</attribute>
+                <attribute name="path">/product_reviews/{id}/apply_transition/accept</attribute>
+                <attribute name="controller">sylius.api.state_machine_transition_applicator:accept</attribute>
+            </itemOperation>
+            <itemOperation name="reject_product_review">
+                <attribute name="method">PATCH</attribute>
```

Comment:
I don't know how configure serialization group to have empty request body in state machine transition paths

Reply:
How does it look right now?

Reply:
![Screenshot 2020-03-05 at 14 58 12](https://user-images.githubusercontent.com/29897151/75988407-d280b100-5ef1-11ea-9c82-0a0ded566580.png)

but as I sad wy should have an empty body

Reply:
It is not possible to update any of these fields according to https://github.com/Sylius/Sylius/pull/11173/files#diff-8705efddce0ceb9c7db5b7c1aa26bdc9R30-R35 (at least I think so). 

Something, that should be reiterated later, but we can accept the current solution right now.